src/vmm/src/rpc_interface.rs

```diff
@@ -514,12 +514,9 @@ impl<'a> PrebootApiController<'a> {
     // On success, this command will end the pre-boot stage and this controller
     // will be replaced by a runtime controller.
     fn load_snapshot(&mut self, load_params: &LoadSnapshotParams) -> ActionResult {
-        warn!(
-            "{} {}",
-            DEV_PREVIEW_LOG_PREFIX, "Restoring snapshots is currently in development preview."
-        );
+        log_dev_preview_warning!("Virtual machine snapshots");
 
-        let load_start_us = utils::time::get_time_us(utils::time::ClockType::Monotonic);
+        let _load_start_us = utils::time::get_time_us(utils::time::ClockType::Monotonic);
```

Comment:
You are right. This is a change made with previous logic where it was possible to be unused which failed compilation. That logic no longer applies, I can change this back.

Reply:
> This looks good!
I would like this change to be introduced by only one commit, instead of 2 (one creating a macro and another one switching from macro to function). It's cleaner this way. You can just squash the two commits into one and modify the commit message accordingly. WDYT?

tl;dr - Sure I don't feel too strongly on this point, I will squash in a new revision.

Both commits take different approaches that I consider logically equivalent and of the same quality which is why I often don't squash commits with these circumstances.